### ISSUE TITLE
Mention registry in the section defining extensions

### ISSUE THREAD

**Issue by dret** [OAI/OpenAPI-Specification#3879]
The registry is not overly full, but it exists and hopefully will grow over time. Should the section defining extensions mention the registry and link to it and encourage people to (a) consider re-using extensions, and/or (b) consider creating reusable extensions? Following the lead of IETF/IANA that would require for registered extensions to be well-defined and well-described, because otherwise reuse is almost impossible.

**Comment by handrews** [comment]
@dret PR #3861 adds links to both the registry and the learn site near the top of the spec, and to the format registry in the data types section.  But I forgot about the extension section, so I'll add that to the PR as well.

**Comment by handrews** [comment]
@dret I have [added a commit](https://github.com/OAI/OpenAPI-Specification/pull/3861/commits/d6b71cdf052636fcbc0a73c2a773d6d8feb32ae1) with this change (and generally rewrote that section a bit as I found its wording a bit hard to follow).

**Comment by dret** [comment]
Looks all good to me and I think we can safely close this issue. Thanks, @handrews!
